lib/backends/sdc/volumes.js

```diff
@@ -133,6 +135,30 @@ function createVolume(volumeParams, options, callback) {
                     });
             }
         },
+        function parseAffinityHints(_, cb) {
+            if (!payload.labels) {
+                cb();
+                return;
+            }
+
+            // This sucks, as I'd prefer DAPI to do this affinity parsing work
```

Comment:
Even if it does, I'd personally go for better wording. Might be kids looking at this in the future

Reply:
Heh - updated for the kiddo's.